### PR TITLE
Specify Content-Type to objects

### PR DESCRIPTION
**Pull request by sorah**

### CODE REVIEW

File: lib/s3_assets_uploader/uploader.rb [hunk 1]
```diff
@@ -41,6 +42,15 @@ def compute_asset_key(path)
       end
     end
 
+    def guess_content_type(path)
+      mime_type = MIME::Types.type_for(path.basename.to_s).first
```
Comment (sorah) [inline lib/s3_assets_uploader/uploader.rb:47]: MIME::Types raises an error for `Pathname` objects :tired_face:

Comment (eagletmt) [inline lib/s3_assets_uploader/uploader.rb:47]: I think `require 'mime/types'` is required in this file.

Comment (sorah) [inline lib/s3_assets_uploader/uploader.rb:47]: :no_mouth: uh oh

**Comment by sorah** [comment]
fixed

**Comment by eagletmt** [comment]
I'm a bit worried about gzipped assets.
Before sprockets 3.0.0, sprockets generates gzipped assets and they would have content type `application/gzip` with this implementation.

**Comment by sorah** [comment]
hm, we have to do detect `.gz` → content-type for extension w/o trailing `.gz` + content-encoding gzip?

I guess both previous and current implementation can't handle gzipped files well.

**Comment by eagletmt** [comment]
> I guess both previous and current implementation can't handle gzipped files well.

Yes.
I'll consider this issue when I encounter such situation.

**Comment by eagletmt** [comment]
Released v0.2.0 :gem:

**Comment by sorah** [comment]
:sparkles:

**Pull request merged**